assets/scripts/segments/Segment.jsx

```diff
@@ -204,9 +207,11 @@ export class Segment extends React.Component {
         // If the shift key is pressed, we remove all segments
         if (event.shiftKey === true) {
           removeAllSegments()
```

Comment:
Does `removeAllSegments()` duplicate the `clearSegments()` action? (Same question for `removeSegment()` and `this.props.removeSegment()`, lines 213, 214)

Reply:
we could probably tidy this up more. 
`removeSegment`  from `remove.js` calls `infoBubble.hideSegment` and more infoBubble stuff. 
I'll look into it.